rio/placer/placers_darwin.go

```diff
@@ -0,0 +1,124 @@
+// +build linux
```

Comment:
this is... not doing what you think it is.  in fact it appears to be doing nothing?  the `*_$GOARCH.go` filename appears to have trumped it.